src/Reveal.Sdk.Dom.Tests/Data/DataSourceItems/GoogleAnalytics4DataSourceItemFixture.cs

```diff
@@ -0,0 +1,74 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Reveal.Sdk.Dom.Core.Extensions;
+using Reveal.Sdk.Dom.Data;
+using Xunit;
+
+namespace Reveal.Sdk.Dom.Tests.Data.DataSourceItems
+{
+    public class GoogleAnalytics4DataSourceItemFixture
+    {
+        [Fact]
+        public void GoogleAnalytics4DataSourceItem_IsDataSourceItem_WhenConstructed()
+        {
+            // Arrange
+            var dataSource = new GoogleAnalytics4DataSource();
+
+            // Act
+            var dataSourceItem = new GoogleAnalytics4DataSourceItem("Test title", dataSource);
+
+            // Assert
+            Assert.True(dataSourceItem is DataSourceItem);
+        }
+
+        [Fact]
+        public void Constructor_SetsTitleAndDatasource_AsProvided()
```

Comment:
Need to separate this into 2 unit tests, one for title and one for Datasource

Reply:
This unit test can remain as a single test because it tests the behavior of the constructor setting both Title and DataSource properties at the same time. The context of the test is focused on the constructor's ability to initialize the object with provided values, and asserting both properties in one test aligns with this purpose.